sites/setup_environment.md

```diff
@@ -0,0 +1,63 @@
+# Setting up the environment
+- [Obtaining all resources](#obtaining-all-resources)
+- [How to load the EPIC environment](#how-to-load-the-epic-environment)
+- [How to load the Python virtual environment](#how-to-load-the-python-virtual-environment)
+
+
+## Obtaining all resources
+Before you start, you must clone this repository to your working directory on Cirrus.
+You can do this by using the command:
+
+```bash
+cd /work/d185/d185/$USER
+git clone https://github.com/EPIC-model/workshop-trieste.git
```

Comment:
`d185`

and do we want to change the repository name?

Reply:
Note: I have already changed the repo name; though I did not do in the markdown files. Done now.